XcodeServerSDK/Server Entities/DeviceSpecification.swift

```diff
@@ -133,7 +139,7 @@ public class DeviceFilter : XcodeServerEntity {
         public static func architectureFromPlatformType(platformType: DevicePlatform.PlatformType) -> ArchitectureType {
             
             switch platformType {
-            case .iOS, .iOS_Simulator, .watchOS, .watchOS_Simulator, .Unknown:
+            case .iOS, .iOS_Simulator, .watchOS, .watchOS_Simulator, .tvOS, .Unknown:
```

Comment:
Oh, can you make sure that's the case from a Bot config? I think the architecture is `0` for iOS-like platforms and something like `7` for Mac. So if tvOS bots have 0 then yes, this is correct (also - missing the `tvOS_Simulator` here as well).